### DISCUSSION TITLE
Hello, will you release a version compatible with jdk8 or jdk11?

### DISCUSSION THREAD

**Discussion by koalsky** [Auties00/WhatsappWeb4j#74] [Ideas]
The JDK version is too high. Many of my projects are based on jdk8 or 11

If I use whatsappweb4j, my project needs to be upgraded to JDK 17

In this way, I will have a lot of work to do.

**Comment by Auties00** [comment]
> The JDK version is too high. Many of my projects are based on jdk8 or 11
> 
> If I use whatsappweb4j, my project needs to be upgraded to JDK 17
> 
> In this way, I will have a lot of work to do.

Upgrading from jfk 11 to 17 should take 5 minutes unless you are in some gigantic project

**Comment by GedMarc** [comment]
no we are on a 6 month security and update cycle across the board, in every language and every product - whether it's java microsoft apple or oracle, that's the world today..

Don't step up if you can't keep up

**Reply by Auties00** [reply]
> no we are on a 6 month security and update cycle across the board, in every language and every product - whether it's java microsoft apple or oracle, that's the world today..
> 
> Don't step up if you can't keep up

Not that stepping up, in this case, takes more than changing a property in your pom, there were no breaking changes from jdk 11 to 17 apart from the strict module system which is not a problem here since we are compliant with Jigsaw. It's like if people were scared to upgrade for some reason

**Comment by koalsky** [comment]
Thank you very much. I agree with you.I will try to accept new things.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Alessandro ***@***.***&gt;; 
发送时间: 2021年12月14日(星期二) 凌晨1:12
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Auties00/WhatsappWeb4j] Hello, will you release a version compatible with jdk8 or jdk11? (Discussion #74)





  
no we are on a 6 month security and update cycle across the board, in every language and every product - whether it's java microsoft apple or oracle, that's the world today..
 
Don't step up if you can't keep up
  
Not that stepping up, in this case, takes more than changing a property in your pom, there were no breaking changes from jdk 11 to 17 apart from the strict module system which is not a problem here since we are compliant with Jigsaw. It's like if people were scared to upgrade for some reason
 
&mdash;
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
